service/application/src/main/java/com/solace/maas/ep/event/management/agent/subscriber/messageProcessors/DynamicResourceConfigurationHelper.java

```diff
@@ -0,0 +1,50 @@
+package com.solace.maas.ep.event.management.agent.subscriber.messageProcessors;
+
+import com.solace.maas.ep.common.model.EventBrokerResourceConfiguration;
+import com.solace.maas.ep.common.model.ResourceConfigurationType;
+import com.solace.maas.ep.event.management.agent.event.MessagingServiceEvent;
+import com.solace.maas.ep.event.management.agent.service.EventBrokerResourceConfigToEventConverter;
+import com.solace.maas.ep.event.management.agent.service.MessagingServiceDelegateServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+
+import java.util.List;
+
+@Component
+@Slf4j
+public class DynamicResourceConfigurationHelper {
+
+    private final MessagingServiceDelegateServiceImpl messagingServiceDelegateService;
+
+    private final EventBrokerResourceConfigToEventConverter eventBrokerResourceConfigToEventConverter;
+
+    public DynamicResourceConfigurationHelper(MessagingServiceDelegateServiceImpl messagingServiceDelegateService,
+                                              EventBrokerResourceConfigToEventConverter eventBrokerResourceConfigToEventConverter) {
+        this.messagingServiceDelegateService = messagingServiceDelegateService;
+        this.eventBrokerResourceConfigToEventConverter = eventBrokerResourceConfigToEventConverter;
+    }
+
+    public void loadSolaceBrokerResourceConfigurations(List<EventBrokerResourceConfiguration> resources) {
+        if (CollectionUtils.isEmpty(resources)) {
+            return;
+        }
+        try {
+
+            List<MessagingServiceEvent> solaceEventBrokerResources = resources.stream()
+                    .filter(resource -> resource.getResourceConfigurationType() == ResourceConfigurationType.SOLACE)
+                    .map(eventBrokerResourceConfigToEventConverter::mapToMessagingServiceEvent)
+                    .toList();
+            messagingServiceDelegateService.addMessagingServices(solaceEventBrokerResources)
+                    .forEach(messagingServiceEntity ->
+                            log.info("Loaded [{}] resource with id: [{}] and name: [{}] from message payload.",
```

Comment:
I think this logLevel can be lowered to `debug`. Also, since this method can be called for a list of 10 brokers, I think it is better for this part to produce 1 log instead of 10.

Reply:
Changed log level to debug. 
Additionally, I don't think we can have more than 1 dynamic broker config (Config push is done against a single broker, so is scan) Moreover, it may make sense to see what broker configurations were added successfully and what broker config was faulty and failed